amq/amq62-persistent-ssl.json

```diff
@@ -578,15 +574,11 @@
                     }
                 ],
                 "replicas": 1,
-                "selector": {
-                    "deploymentConfig": "${APPLICATION_NAME}-drainer"
-                },
                 "template": {
                     "metadata": {
                         "name": "${APPLICATION_NAME}-drainer",
                         "labels": {
-                            "deploymentConfig": "${APPLICATION_NAME}-drainer",
-                            "application": "${APPLICATION_NAME}"
+                            "application": "${APPLICATION_NAME}-drainer"
```

Comment:
@rcernich @spolti If I'm getting the logic for listing resources created by single template, here should also be "${APPLICATION_NAME}", right? And with it probably some selector as component=drainer, so two deploymentconfigs do not create selectors with same label...

Reply:
correct